clang/include/clang/DPCT/DPCTOptions.inc

```diff
@@ -372,6 +372,9 @@ DPCT_ENUM_OPTION(DPCT_OPT_TYPE(static llvm::cl::bits<ExperimentalFeatures>), Exp
         false),
              DPCT_OPT_ENUM("user-defined-reductions", int(ExperimentalFeatures::Exp_UserDefineReductions),
         "Experimental extension that allows user define reductions.\n",
+        false),
+             DPCT_OPT_ENUM("async-dpl-algorithm", int(ExperimentalFeatures::Exp_AsyncDPLAlgorithm),
```

Comment:
Let's use one option to control using of APIs from oneapi::dpl::experimental for dpl.  so please update the option name and desc.